gamemodes/terrortown/gamemode/client/cl_help/populate_administration.lua

```diff
@@ -0,0 +1,103 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/administration")
+
+local function PopulateHUDPanel(parent)
+	local form = CreateForm(parent, "header_hud_administration")
+
+	local restrictedHUDs = ttt2net.GetGlobal({"hud_manager", "restrictedHUDs"})
+	local hudList = huds.GetList()
+	local validHUDsDefault = {}
+	local validHUDsRestriction = {[1] = "None"}
+
+	for i = 1, #hudList do
+		validHUDsDefault[#validHUDsDefault + 1] = hudList[i].id
+		validHUDsRestriction[#validHUDsRestriction + 1] = hudList[i].id
+	end
+
+	form:MakeHelp({
+		label = "help_hud_default_desc"
+	})
+
+	form:MakeComboBox({
+		label = "label_hud_default",
+		choices = validHUDsDefault,
+		selectName = ttt2net.GetGlobal({"hud_manager", "defaultHUD"}) or "None",
+		default = "None",
+		OnChange = function(_, _, value)
+			net.Start("TTT2DefaultHUDRequest")
+			net.WriteString(value == "None" and "" or value)
+			net.SendToServer()
+		end
+	})
+
+	form:MakeHelp({
+		label = "help_hud_forced_desc"
+	})
+
+	form:MakeComboBox({
+		label = "label_hud_force",
+		choices = validHUDsRestriction,
+		selectName = ttt2net.GetGlobal({"hud_manager", "forcedHUD"}) or "None",
+		default = "None",
+		OnChange = function(_, _, value)
+			net.Start("TTT2ForceHUDRequest")
+			net.WriteString(value == "None" and "" or value)
+			net.SendToServer()
+		end
+	})
+
+	local form2 = CreateForm(parent, "header_hud_enabled")
+
+	form2:MakeHelp({
+		label = "help_hud_enabled_desc"
+	})
+
+	for i = 1, #hudList do
+		local hud = hudList[i]
+
+		form2:MakeCheckBox({
+			label = hud.id,
+			initial = not table.HasValue(restrictedHUDs, hud.id),
+			default = true,
+			OnChange = function(_, value)
+				net.Start("TTT2RestrictHUDRequest")
+				net.WriteString(hud.id)
+				net.WriteBool(not value)
+				net.SendToServer()
+			end
+		})
+	end
+end
+
+ttt2net.OnUpdateGlobal({"hud_manager", "restrictedHUDs"}, function()
+	if HELPSCRN:GetOpenMenu() ~= "ttt2_administration_hud" then return end
+
+	-- rebuild the content area so that data is refreshed
+	-- based on the newly restricted HUDs
+	vguihandler.Rebuild()
+end)
+
+local function PopulateRandomShopPanel(parent)
+
```

Comment:
Yes. LeBroomer will add this in a future PR and I added this empty menu as a placeholder for him. I talked with tkindanight about this and we decided on keeping the empty menues since they will be filled asap.
If you really dislike it, I can remove it and readd in a new PR.

Reply:
Probably @lebroomer could create a trello entry, then I'm fine with it :)